packages/markdown/src/from-portable-text/renderers/type.ts

```diff
@@ -38,7 +39,7 @@ export const DefaultImageRenderer: PortableTextTypeRenderer<{
   alt: string | undefined
   title: string | undefined
 }> = ({value}) => {
-  const alt = value.alt ?? ''
+  const alt = escapeMarkdown(value.alt ?? '')
   const title = value.title ? ` "${value.title}"` : ''
```

Comment:
Minor: we  may need to escape titles here too? 

Would a `value.title` of `path\to "file"` break these links?